docs/user/sampling.md

```diff
@@ -0,0 +1,67 @@
+# Sampling
+
+`LocalRunner` gets trajectories through Sampling to train the policy. In
+Garage, [`LocalRunner`](https://github.com/rlworkgroup/garage/blob/a767dcb95988abc14165a2a320ee39932a1c85ca/src/garage/experiment/local_runner.py)
+uses [`Sampler`](https://github.com/rlworkgroup/garage/blob/a767dcb95988abc14165a2a320ee39932a1c85ca/src/garage/sampler/sampler.py)
+to perform sampling. `Sampler` manages [`Worker`](https://github.com/rlworkgroup/garage/blob/a767dcb95988abc14165a2a320ee39932a1c85ca/src/garage/sampler/worker.py#L5)s
+and assign specific tasks to them, such as performing rollouts and update
+agents and environments.
+
+## Sampler
+
+Sampler is responsible for assign sampling jobs to workers. Garage now has
+three types of `Sampler`s:
+
+- [`LocalSampler`](https://github.com/rlworkgroup/garage/blob/a767dcb95988abc14165a2a320ee39932a1c85ca/src/garage/sampler/local_sampler.py),
+the default sampler, which runs workers in the main process. With this sampler,
+all the sampling tasks will be run in the same thread.
+
+- [`MultiprocessingSampler`](https://github.com/rlworkgroup/garage/blob/master/src/garage/sampler/multiprocessing_sampler.py),
```

Comment:
You probably shouldn't put `MultiprocessingSampler` here. It currently only works with PyTorch CPU. Once I make it work with more backends I'll add it to this document.